crates/model/src/interaction.rs

```diff
@@ -0,0 +1,10 @@
+use primitive_types::{H160, U256};
+use serde::{Deserialize, Serialize};
+
+#[derive(Eq, PartialEq, Clone, Debug, Hash, Default, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct InteractionData {
+    pub to: H160,
```

Comment:
Should we call this `target` for consistency with the Solidity structure:

https://github.com/cowprotocol/contracts/blob/d043b0bfac7a09463c74dfe1613d0612744ed91c/src/contracts/libraries/GPv2Interaction.sol#L9-L13